js/src/chrome/projects/chrome.projects.js

```diff
@@ -1255,7 +1257,7 @@ Wu.Chrome.Projects = Wu.Chrome.extend({
 		}
 
 		// slack
-		this._logUpdate({
+		!window.testMode && this._logUpdate({
```

Comment:
This can be moved to the `app.log()` function, so you dont have to put `if (window.testMode)` in so many places. Better to put them in the most common function, ie. `app.log()`